sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -53,6 +54,8 @@ export class SchemaRegistryClient implements SchemaRegistry {
     readonly fullyQualifiedNamespace: string;
     getSchema(schemaId: string, options?: GetSchemaOptions): Promise<Schema>;
     getSchemaProperties(schema: SchemaDescription, options?: GetSchemaPropertiesOptions): Promise<SchemaProperties>;
+    // (undocumented)
+    pipeline: Pipeline;
```

Comment:
Did you discuss this change with our architects? if not, I would say it warrants a mini archboard meeting to discuss it.

For now, I think we can access it without exposing it using `[]` operator.

Reply:
Referring to the discussion here https://github.com/Azure/azure-sdk-for-js/issues/16876.

@jeremymeng @xirzec @mikeharder @bterlson @ramya-rao-a

Reply:
Was there any follow-ups on what was discussed in the github issue? The discussion does not seem to have a conclusion on what to do or how.

Reply:
No conclusion, hence tagged everyone here as well 🙂

Reply:
It feels a little odd to expose it only for test purposes, but perhaps we need to reserve time (next Wednesday maybe?) to talk about if the pattern should be adopted more broadly? I know several SDKs have shipped with it exposed and it hasn't caused *too* many problems. I remember in one case there was a bit of a naming collision, though.

Reply:
Removed pipeline attribute and instead passed client["client"] that hosts the pipeline to the configure client method.
I don't like the approach mainly because it's not necessary that all SDKs have written the clients the same way(?).

However, I'll bring this up next week in the team meeting for opinions. 
This #16876 will act as the tracking issue.